packages/web/src/scss/components/Tabs/index.html

```diff
@@ -52,7 +51,62 @@ <h2 class="docs-Heading">Default</h2>
     </div>
 
   </div>
+</section>
+
+<section class="UNSTABLE_Section">
+  <h2 class="docs-Heading">Custom Spacing</h2>
+
+  <div class="docs-Stack docs-Stack--stretch">
 
+    <ul class="Tabs"
+        role="tablist"
+        style="--tabs-spacing: var(--spirit-space-400); --tabs-spacing-tablet: var(--spirit-space-800); --tabs-spacing-desktop: var(--spirit-space-1200)"
+    >
+      <li class="Tabs__item">
+        <button
+          class="Tabs__link is-selected"
+          role="tab"
+          type="button"
+          aria-selected="true"
+          id="pane-3-tab"
+          data-spirit-toggle="tabs"
+          data-spirit-target="#pane-1"
```

Comment:
this is changing the first Tabs :) please change it to the correct pane. 

Maybe recheck all the IDs in this file/section.